webhooks-extension/cmd/extension/README.md

```diff
@@ -55,6 +55,24 @@ Example POST
 }
 ```
 
+### DELETE endpoint
+
+```
+DELETE /webhooks/<webhookid>?namespace=<my namespace>
```

Comment:
interesting design question here - do we want namespace as a query param or as a path param ..... namespace/{namespace}/webhooks/{webhookid} ...... although we quite like having the /webhooks start ... so maybe /webhooks/namespace/{namespace}/webhooks/{webhook} although that seems long winded ... maybe query param is the nicest.

we get into quite a deep conversation about differences between the dashboard and the extension ... the dashboard being designed so that you choose a namespace and then all controls are on that namespace .... vs the extension taking namespace as selection in many places.  Possibly we should discuss as a wider group

Reply:
I'll leave in the query namespace for now for future proofing, we can behave a little differently from the main dashboard with our own API definitions and, like you, I'm quite a fan of the namespace being a query parameter, so I'm going to leave it as is. I don't really want to change everything to be, for example, /v1/namespaced, at least not yet, until we have a proper namespace and consistency story going forward